Makefile

```diff
@@ -293,57 +291,49 @@ release-notes: $(RELEASE_NOTES)
 ## --------------------------------------
 
 .PHONY: create-cluster
-create-cluster: ## Create a development Kubernetes cluster on Azure in a KIND management cluster.
+create-cluster: $(CLUSTERCTL) $(ENVSUBST) ## Create a development Kubernetes cluster on Azure in a KIND management cluster.
 	kind create cluster --name=clusterapi
 
-	# Install cert manager.
-	kubectl \
-		create -f examples/_out/cert-manager.yaml
-	# Wait for webhook servers to be ready to take requests
-	kubectl \
-		wait --for=condition=Available --timeout=5m apiservice v1beta1.webhook.cert-manager.io
+	# Install cert manager and wait for availability
+	kubectl create -f https://github.com/jetstack/cert-manager/releases/download/v0.11.1/cert-manager.yaml
+	kubectl wait --for=condition=Available --timeout=5m apiservice v1beta1.webhook.cert-manager.io
 
-	# Apply provider-components.
-	kubectl \
-		create -f examples/_out/provider-components.yaml
-	# Wait for capi-controller
-	kubectl \
-		wait --for=condition=Ready --timeout=5m -n capi-system pod -l control-plane=controller-manager
-    # Wait for capz-controller
-	kubectl \
-		wait --for=condition=Ready --timeout=5m -n capz-system pod -l control-plane=capz-controller-manager
+	# Deploy CAPI
+	kustomize build github.com/kubernetes-sigs/cluster-api//config?ref=v0.3.0-rc.2 | kubectl apply -f -
+	kustomize build github.com/kubernetes-sigs/cluster-api//bootstrap/kubeadm/config?ref=v0.3.0-rc.2 | kubectl apply -f -
+	kustomize build github.com/kubernetes-sigs/cluster-api//controlplane/kubeadm/config?ref=v0.3.0-rc.2 | kubectl apply -f -
+
+	# Deploy CAPZ
+	kustomize build config | $(ENVSUBST) | kubectl apply -f -
+
+	# Wait for CAPI pods
+	kubectl wait --for=condition=Ready --timeout=5m -n capi-system pod -l cluster.x-k8s.io/provider=cluster-api
+	kubectl wait --for=condition=Ready --timeout=5m -n capi-kubeadm-bootstrap-system pod -l cluster.x-k8s.io/provider=bootstrap-kubeadm
+	kubectl wait --for=condition=Ready --timeout=5m -n capi-kubeadm-control-plane-system pod -l cluster.x-k8s.io/provider=control-plane-kubeadm
+
+	# Wait for CAPZ pods
+	kubectl wait --for=condition=Ready --timeout=5m -n capz-system pod -l cluster.x-k8s.io/provider=infrastructure-azure
+	
 	# Create Cluster.
 	sleep 10
+	kustomize build templates | $(ENVSUBST) | kubectl apply -f -
```

Comment:
You can probably skip the kustomize here, we mainly needed it in CAPA because we make modifications to it for the conformance tests we have in place.